### PR TITLE
paymentmethod-smoke

### PR DESCRIPTION
**Pull request by infanta-deriv**
Card - https://app.clickup.com/t/20696747/DPROD-3812 , https://app.clickup.com/t/20696747/DPROD-3813 
This PR has script for homepage - Payment methods for ROW and EU

### CODE REVIEW

**Review by rajendra-deriv — APPROVED** [review]
Looks good to me

File: cypress/e2e/smoke/allHomePayments.cy.js [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+      paymentMethodListEU.forEach((method) => {
+      cy.findAllByRole('img', { name: method }).first().should('exist')
+    }) 
+  }
     cy.findByTestId('fast-payment-link').click()
     cy.url().should('include', '/payment-methods/')
+    cy.findByRole('heading', { name: 'Payment methods' }).should('be.visible')
 }
 
 describe('Validate payment methods section in home page', () => {
```
Comment (meeyun-deriv) [inline cypress/e2e/smoke/allHomePayments.cy.js:27]: can you also help to add the test case custom ID?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**